config/superglue-bert.conf

```diff
@@ -8,7 +8,6 @@ exp_name = "bert-large-cased"
 max_seq_len = 256 // Mainly needed for MultiRC, to avoid over-truncating
                   // But not 512 as that is really hard to fit in memory.
 tokenizer = "bert-large-cased"
-
```

Comment:
Restore?

Reply:
Not sure why it's appearing in github as removed because it isn't